reledmac.dtx

```diff
@@ -7107,7 +7107,7 @@
 % 
 % \begin{macro}{\@beforeinsertofthisedtext}
 % \cs{@beforeinsertofthisedtext} is an internal macro. \macpackage or \parpackage can add in this macro any content required to be executed before doing any \cs{insert} related to a \cs{edtext}.
-% It content is let to \cs{relax} at the end of every \cs{edtext}.
+% Its content is added to \cs{relax} at the end of every \cs{edtext}.
```

Comment:
My esteemed colleague, I do not wish to argue with such a expert as yourself. However, I have the TeX Book and do not see such a use of the phrase as “a command let . . . etc.” Usually, in computer lingo, “let” is used something like this: “let A equal 50 and B equal 33”.

Nevertheless, I yield to your knowledge.

M


> On 2016-07-01, at 03:55 , Maïeul Rouquette <notifications@github.com> wrote:
> 
> In reledmac.dtx <https://github.com/maieul/ledmac/pull/551#discussion_r69262429>:
> 
> > @@ -7107,7 +7107,7 @@
> >  % 
> >  % \begin{macro}{\@beforeinsertofthisedtext}
> >  % \cs{@beforeinsertofthisedtext} is an internal macro. \macpackage or \parpackage can add in this macro any content required to be executed before doing any \cs{insert} related to a \cs{edtext}.
> > -% It content is let to \cs{relax} at the end of every \cs{edtext}.
> > +% Its content is added to \cs{relax} at the end of every \cs{edtext}.
> no! no ! "a command let to an other command" is a typicall TeX construction.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/551/files/c706e111b6e784782454e37324d1239e0378858b#r69262429>, or mute the thread <https://github.com/notifications/unsubscribe/AHG-QDfG0AMjVguA7zdbOcfYueo9s1drks5qRMf4gaJpZM4JC5jq>.
>